### PR TITLE
fix : Purged Unwanted Imports in BackEndApp

### LINKED ISSUE #320: Purge Unwanted Imports in BackEndApp
## Feature Request

Is your feature request related to an existing [Issue](https://www.github.com/Auto-DL/Generator/issues) or a new implementation?

No

### Feature Description

Many files present in the `BackEndApp` consist of many unused / unwanted imports.

### Possible Solution

Remove the non-essential imports from the `BackEndApp`.
This will help the codebase remain clean and concise! 😄

### PR DESCRIPTION
**Pull request by a-sambhab**
# Pull Request
Removed the non-essential imports from the BackEndApp

## What does this PR do?
Fixes #320 

 Please include a summary of the change and which issue is fixed.
 Removed all the non-essential imports from the BackEndApp to make sure that the code is clean and concise

 If we didn't approve off or discuss your PR in Github issues or slack there's a high chance it will not be merged.

The following links the related issue to the PR (https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
-->

## What part does this affect?
- [ ] FrontEnd.
- [x] BackEnd.
- [ ] Documentation.
- [ ] Other. (Please specify below)

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue or slack?
- [x] Did you read the [contributor guideline](https://github.com/Auto-DL/Auto-DL/blob/v1-beta/CONTRIBUTING.md)?
- [x] Did you ensure that there aren't any other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change?
- [x] Did you make sure the title is self-explanatory and the description concisely explains the PR?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure the code is clean and docstrings have been added or updated as required?
- [x] Did you make sure the code is linted/formatted locally prior to submission? (using `black` and/or `prettier`)
- [x] Did you make sure to update the documentation with your changes? (if necessary)

## PR review
Anyone in the community is free to review the PR once the tests have passed.

Thank you for contributing to [AutoDL](https://github.com/Auto-DL/Auto-DL). We look forward to your continued support.

### CODE REVIEW

File: BackEndApp/v1/urls.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 from os import name
-from django.contrib import admin
 from django.urls import path, include
```
Comment (McTechie) [inline BackEndApp/v1/urls.py:2]:
```suggestion
from django.urls import path
```

**Review by McTechie — CHANGES_REQUESTED** [review]
It would be great if the files from FrontEndApp (specifically `package-lock.json` and `mockServiceWorker.js`) would not be pushed as the PR is catered towards the `BackEndApp`.

P.s. You could keep the files as is by grabbing the changes from upstream repo:
```
git checkout upstream/main -- <file-name-including-path>
```
\- followed by committing and pushing the same xD

**Comment by a-sambhab** [comment]
hey @McTechie im not able to get the use of the above git command can you somehow help me out if possible??

**Comment by McTechie** [comment]
Ahh you could try using the commands below:
1. `git remote -v` : To list the git remotes locally (If you have added the Upstream and are able to see upstream repository urls, skip to 3).
2. `git remote add upstream https://github.com/Auto-DL/Auto-DL.git` : To set the upstream repository (the one which you're creating a PR to xD)
3. `git checkout upstream/main -- <file-name-including-path>` : To directly get the files present in the upstream repo.

Use the last command for files `package-lock.json` and `mockServiceWorker.js` (by replacing their paths correctly in the command) to revert the changes pushed in this PR (for those files only).

Once you do these you will have to commit and push the changes one last time. Hope this helps! xD

**Comment by a-sambhab** [comment]
Hey, @McTechie can you review again please.

**Comment by McTechie** [comment]
Ahh @a-sambhab there appears to be additional files added in this PR (not catered to the Issue - Purge Unwanted imports in `BackEndApp`). The commands above were to remove the additional files from the PR, but ig some other changes were done and pushed as well. 👀 

Additionally there happens to be a merge conflict (should be resolved before pushing 😅).

Suggestion: Create a new PR and make sure that only the `BackEndApp` files are pushed.

@ADI10HERO any thoughts on this...??

**Comment by a-sambhab** [comment]
@McTechie ill be sending a new pr, this is actually new to me so I'm sorry as well 😔, so I'm closing this pr

**Comment by McTechie** [comment]
No issues at all. 😄
Just make sure to add, commit and push the files altered by you in `BackEndApp` ✨

**Pull request closed without merging**